### ISSUE TITLE
No working mount applet on Xiaomi MI A1 @ 8.1.0

### ISSUE THREAD

**Issue by d4rken** [d4rken/sdmaid-public#3325]
```
1582414554948  I/Debug:ApkInfo: F: VERSIONNAME: 4.15.6, VERSIONCODE: 41506
1582414554947  D/Debug:BuildProp: Fingerprint: xiaomi/tiffany/tiffany:8.1.0/OPM1.171019.019/V11.0.3.0.ODBCNXM:user/release-keys
1582414554949  V/RXS:RxCmdShell: isAlive()
1582414554949  D/Debug:BuildProp: ro.build.version.codename=REL
1582414554950  I/Debug:ApkInfo: P: VERSIONNAME: 4.4.0, VERSIONCODE: 40400
1582414554950  D/Debug:BuildProp: ro.build.version.incremental=V11.0.3.0.ODBCNXM
1582414554947  D/Debug:SysInfoModule: App locales: [tr_TR], Device locales: [tr_TR]
1582414554951  V/RXS:RxCmdShell: open()
1582414554951  D/Debug:BuildProp: ro.build.version.base_os=
1582414554951  D/Debug:SysInfoModule: isArmed: true
1582414554952  D/Debug:BuildProp: ro.build.version.release=8.1.0
1582414554952  D/Debug:BuildProp: ro.build.display.id=xiaomi.eu
1582414554953  D/Debug:BuildProp: ro.product.name=tiffany
1582414554953  V/RXS:RxShell: open()
1582414554954  D/Debug:BuildProp: ro.product.device=tiffany
1582414554954  V/RXS:RxProcess: open()
1582414554954  D/Debug:BuildProp: ro.product.board=msm8953
1582414554955  D/Debug:BuildProp: ro.product.manufacturer=Xiaomi
1582414554955  D/Debug:BuildProp: ro.product.brand=xiaomi
1582414554956  D/Debug:BuildProp: ro.product.model=MI A1
```

```
1582414566117  D/RXS:RxShell:Session: writeLine(line=/data/user/0/eu.thedarken.sdm/files/toybox_sdm mount -o ro,remount '/system', flush=false)
1582414566117  D/RXS:RxShell:Session: writeLine(line=echo 9b122910-f26b-4307-85cd-191eac21132f $?, flush=false)
1582414566117  D/RXS:RxShell:Session: writeLine(line=echo 9b122910-f26b-4307-85cd-191eac21132f >&2, flush=true)
1582414566121  V/RXS:Harvester:Error: mount: '/system' not in /proc/mounts
```

```
1582414570174  V/RXS:Harvester:Output: rootfs on / type rootfs (rw,seclabel)
1582414570175  V/RXS:Harvester:Output: /dev/root on / type ext4 (ro,seclabel,relatime,data=ordered)
1582414570175  V/RXS:Harvester:Output: tmpfs on /sbin type tmpfs (rw,seclabel,relatime,size=1821852k,nr_inodes=455463,mode=755)
1582414570176  V/RXS:Harvester:Output: /sbin/.magisk/block/persist on /sbin/.magisk/mirror/persist type ext4 (rw,seclabel,relatime,data=ordered)
1582414570176  V/RXS:Harvester:Output: /sbin/.magisk/block/system_root on /sbin/.magisk/mirror/system_root type ext4 (ro,seclabel,relatime,data=ordered)
1582414570176  V/RXS:Harvester:Output: /sbin/.magisk/block/system_root on /sbin/charger type ext4 (ro,seclabel,relatime,data=ordered)
1582414570176  V/RXS:Harvester:Output: /sbin/.magisk/block/system_root on /sbin/restart.sh type ext4 (ro,seclabel,relatime,data=ordered)
1582414570176  V/RXS:Harvester:Output: /sbin/.magisk/block/data on /sbin/.magisk/mirror/data type ext4 (rw,seclabel,relatime,noauto_da_alloc,data=ordered)
1582414570177  V/RXS:Harvester:Output: /sbin/.magisk/block/data on /sbin/.magisk/modules type ext4 (rw,seclabel,relatime,noauto_da_alloc,data=ordered)
1582414570177  V/RXS:Harvester:Output: tmpfs on /dev type tmpfs (rw,seclabel,nosuid,relatime,size=1821852k,nr_inodes=455463,mode=755)
1582414570177  V/RXS:Harvester:Output: devpts on /dev/pts type devpts (rw,seclabel,relatime,mode=600)
1582414570177  V/RXS:Harvester:Output: none on /dev/cpuctl type cgroup (rw,relatime,cpu)
1582414570177  V/RXS:Harvester:Output: adb on /dev/usb-ffs/adb type functionfs (rw,relatime)
1582414570177  V/RXS:Harvester:Output: proc on /proc type proc (rw,relatime,gid=3009,hidepid=2)
1582414570177  V/RXS:Harvester:Output: sysfs on /sys type sysfs (rw,seclabel,relatime)
1582414570177  V/RXS:Harvester:Output: selinuxfs on /sys/fs/selinux type selinuxfs (rw,relatime)
1582414570178  V/RXS:Harvester:Output: debugfs on /sys/kernel/debug type debugfs (rw,seclabel,relatime)
1582414570178  V/RXS:Harvester:Output: tracefs on /sys/kernel/debug/tracing type tracefs (rw,seclabel,relatime)
1582414570178  V/RXS:Harvester:Output: pstore on /sys/fs/pstore type pstore (rw,seclabel,relatime)
1582414570178  V/RXS:Harvester:Output: none on /sys/fs/cgroup type tmpfs (rw,seclabel,relatime,size=1821852k,nr_inodes=455463,mode=750,gid=1000)
1582414570178  V/RXS:Harvester:Output: none on /sys/fs/cgroup/freezer type cgroup (rw,relatime,freezer)
1582414570178  V/RXS:Harvester:Output: none on /sys/fs/cgroup type tmpfs (rw,seclabel,relatime,size=1821852k,nr_inodes=455463,mode=750,gid=1000)
1582414570178  V/RXS:Harvester:Output: none on /sys/fs/cgroup/freezer type cgroup (rw,relatime,freezer)
1582414570179  V/RXS:Harvester:Output: none on /acct type cgroup (rw,relatime,cpuacct)
1582414570179  V/RXS:Harvester:Output: tmpfs on /mnt type tmpfs (rw,seclabel,relatime,size=1821852k,nr_inodes=455463,mode=755,gid=1000)
1582414570179  V/RXS:Harvester:Output: /data/media on /mnt/runtime/default/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid)
1582414570179  V/RXS:Harvester:Output: /data/media on /mnt/runtime/read/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,multiuser,mask=23,derive_gid)
1582414570179  V/RXS:Harvester:Output: /data/media on /mnt/runtime/write/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,multiuser,mask=7,derive_gid)
1582414570180  V/RXS:Harvester:Output: /dev/block/vold/public:179,65 on /mnt/media_rw/45E4-09F9 type vfat (rw,dirsync,nosuid,nodev,noexec,noatime,uid=1023,gid=1023,fmask=0007,dmask=0007,allow_utime=0020,codepage=437,iocharset=iso8859-1,shortname=mixed,utf8,errors=remount-ro)
1582414570180  V/RXS:Harvester:Output: /mnt/media_rw/45E4-09F9 on /mnt/runtime/default/45E4-09F9 type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,mask=6)
1582414570180  V/RXS:Harvester:Output: /mnt/media_rw/45E4-09F9 on /mnt/runtime/read/45E4-09F9 type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,mask=18)
1582414570180  V/RXS:Harvester:Output: /mnt/media_rw/45E4-09F9 on /mnt/runtime/write/45E4-09F9 type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=9997,mask=18)
1582414570181  V/RXS:Harvester:Output: none on /config type configfs (rw,relatime)
1582414570181  V/RXS:Harvester:Output: /dev/block/mmcblk0p49 on /data type ext4 (rw,seclabel,nosuid,nodev,noatime,noauto_da_alloc,data=ordered)
1582414570182  V/RXS:Harvester:Output: /dev/block/mmcblk0p27 on /persist type ext4 (rw,seclabel,nosuid,nodev,noatime,data=ordered)
1582414570183  V/RXS:Harvester:Output: /dev/block/mmcblk0p1 on /firmware type vfat (ro,context=u:object_r:firmware_file:s0,relatime,uid=1000,gid=1000,fmask=0337,dmask=0227,codepage=437,iocharset=iso8859-1,shortname=lower,errors=remount-ro)
1582414570183  V/RXS:Harvester:Output: /dev/block/mmcblk0p13 on /dsp type ext4 (ro,seclabel,nosuid,nodev,relatime,data=ordered)
1582414570183  V/RXS:Harvester:Output: tmpfs on /storage type tmpfs (rw,seclabel,relatime,size=1821852k,nr_inodes=455463,mode=755,gid=1000)
1582414570183  V/RXS:Harvester:Output: /data/media on /storage/emulated type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,multiuser,mask=6,derive_gid)
1582414570184  V/RXS:Harvester:Output: /mnt/media_rw/45E4-09F9 on /storage/45E4-09F9 type sdcardfs (rw,nosuid,nodev,noexec,noatime,fsuid=1023,fsgid=1023,gid=1015,mask=6)
1582414570184  V/RXS:Harvester:Output: tmpfs on /storage/self type tmpfs (rw,seclabel,relatime,size=1821852k,nr_inodes=455463,mode=755,gid=1000)
```

[sdmaid_logfile_1582414553937.txt.zip](https://github.com/d4rken/sdmaid-public/files/4241801/sdmaid_logfile_1582414553937.txt.zip)

**Comment by d4rken** [comment]
#2110 
#2135 
#2903

**Comment by andheartman** [comment]
Sorun fixed beta 4.15.7 versiyon. i am very happy :)

**Comment by d4rken** [comment]
:ok_hand: Scheduled to be included in the next beta release.
